README.md

```diff
@@ -5,18 +5,27 @@ PyDentity Wallet is a Python-based Progressive Web Application (PWA) designed to
 
 ## Features
 - **Multi-Tenant ACA-Py Support**: Connects to ACA-Py instances and allows users to manage subwallets.
-- **Mobile Device Binding**: Securely stores wallet data locally on mobile devices.
+- **Mobile Device Binding**: Securely stores wallet data in an ecrypted remote session bound to a mobile device.
 - **Cross-Platform Compatibility**: Works on iOS and Android via browser-based access.
 - **Open Standards Compliance**: Utilizes W3C, IETF, DIF, and other open standards whenever possible.
 
 ## Project Status
 PyDentity Wallet is currently in the **Labs** stage under the OpenWallet Foundation. The project is actively developed as an open-source initiative to enhance the usability of decentralized identity solutions.
 
 ## Usage
+### Pre requisite
+Before deploying the PyDentity Wallet app, you must have a multitenant acapy instance available. There are example configurations in this repository, otherwise refer to the official acapy documentation.
```

Comment:
The `.env` file specifies an api-key and endpoint: do we actually need a Traction instance, which adds api-keys, rather than a vanilla multitenant agent?

Reply:
This is currently designed to work with a multitenant agent. The api key is the admin api key required to create subwallets. Instance calls are made using wallet_id / wallet_key to request a token.

Reply:
I can rename to `agent_admin_endpoint` and `agent_admin_api_key`